### DISCUSSION TITLE
编译问题

### DISCUSSION THREAD

**Discussion by KuGouGo** [vernesong/OpenClash#2697] [General]
openwrt官方编译过程中会出现不会勾选luci-compat的情况，导致编译后无法正常运行

**Comment by vernesong** [comment]
没有强制依赖，要自己勾

**Reply by rrfeng** [reply]
> 没有强制依赖，要自己勾

@vernesong 
所以如果要自己编译 openwrt，哪些东西是必须要编译时选择的？我记得 kmod-tun 好像直接安装是不行的，只能编译进去。

还有其他的依赖是这样的吗？如果可以安装 ipk 支持的，就可以不用编译选择了。
